website/blog/2025-09-25-hacktoberfest.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Hacktoberfest 2025! - Contribute to Podman Desktop
+description: Contribute to Podman Desktop this Hacktoberfest!
+slug: hacktoberfest-2025
+authors: [rujutashinde]
+tags: [podman-desktop, hacktoberfest]
```

Comment:
```suggestion
tags: [podman-desktop, hacktoberfest, community]
```

Reply:
added community tag
updated contributions paragraph to add cncf and 3m downloads mentions